File: dotnet/system/workspace/typescript/libs/workspace/domain/json/system/src/lib/database/push/pushresult.ts
import { PushResponse } from '@allors/protocol/json/system';
import { IPushResult, ISession } from '@allors/workspace/domain/system';
import { Result } from '../Result';

export class PushResult extends Result implements IPushResult {
  constructor(session: ISession, response: PushResponse) {
    super(session, response);
  }
}

File: dotnet/system/workspace/csharp/adapters/allors.workspace.adapters.remote/database/push/pushresult.cs
// <copyright file="RemotePushResult.cs" company="Allors bvba">
// Copyright (c) Allors bvba. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Workspace.Adapters.Remote
{
    using Allors.Protocol.Json.Api;

    public class PushResult : Result, IPushResult
    {
        public PushResult(ISession session, Response response) : base(session, response)
        {
        }
    }
}
